torch/_inductor/mkldnn.py

```diff
@@ -314,6 +314,12 @@ def pack_module(gm: torch.fx.GraphModule):
                     or dynamo_config.dynamic_shapes
                     or len(node.args) > 1
                     or len(node.kwargs) > 0
+                    or any(
+                        output_padding >= stride
```

Comment:
This won't work with dynamic shapes.  You will just get `TypeError: cannot determine truth value of Relational`
```
>>> 10 > sympy.Symbol("x")
x < 10
```